media-gfx/phototonic/phototonic-2.1.recipe

```diff
@@ -0,0 +1,98 @@
+SUMMARY="An image viewer and organizer built with Qt and Exiv2"
+DESCRIPTION="Features
```

Comment:
I'm not sure about this, but I think we need to mention the name of the app in `DESCRIPTION`.
So, **_maybe_** we could change this to:
```diff
-DESCRIPTION="Features
+DESCRIPTION="PhotoTonic features:
```
Any opinions about this?